lib/conjur/host_factory_token.rb

```diff
@@ -66,5 +66,13 @@ def cidr
     def revoke
       Conjur::API.revoke_host_factory_token @credentials, token
     end
+
+    def ==(other)
```

Comment:
This was an attempt to fix the failing host factory token scenario above. It's a sensible change, but wasn't sufficient to fix the problem.